src/main/proto/ga4gh/rna_quantification_service.proto

```diff
@@ -0,0 +1,147 @@
+  string rna_quantification_id = 1;
+
+  // The `Dataset` to search.
+  string dataset_id = 2;
+
+  // Specifies the maximum number of results to return in a single page.
+  // If unspecified, a system default will be used.
+  int32 page_size = 3;
+
+  // The continuation token, which is used to page through large result sets.
+  // To get the next page of results, set this parameter to the value of
+  // 'nextPageToken' from the previous response.
+  string page_token = 4;
+}
+
+// This is the response from 'POST /rnaquantification/search' expressed as JSON.
+message SearchRnaQuantificationsResponse {
+  // The list of matching quantifications.
+  repeated RnaQuantification rna_quantifications = 1;
+
+  // The continuation token, which is used to page through large result sets.
+  // To get the next page of results, set this parameter to the value of
+  // 'nextPageToken' from the previous response.
+  string next_page_token = 2;
+}
+
+// This request maps to the URL `GET /rnaquantification/{rna_quantification_id}`.
+
+message GetRnaQuantificationRequest {
+  // The ID of the `RnaQuantification`.
+  string rna_quantification_id = 1;
+}
+
+// ****************  /expressionlevel/search  *******************
+// This request maps to the body of 'POST /expressionlevel/search'
+// as JSON.
+message SearchExpressionLevelsRequest {
+  // If present, return matching Expression Level record.
+  string expression_level_id = 1;
+
+  // If present return only ExpressionLevel records which belong to this set.
+  string quantification_group_id = 2;
+
+  // The rnaQuantification to restrict search to.
+  string rna_quantification_id = 3;
+
+  // Only return ExpressionLevel records with expressions exceeding
+  // this value.  (Defaults to 0.0)
+  float threshold = 4;
+
+  // Specifies the maximum number of results to return in a single page.
+  // If unspecified, a system default will be used.
+  int32 page_size = 5;
+
+  // The continuation token, which is used to page through large result sets.
+  // To get the next page of results, set this parameter to the value of
+  // 'nextPageToken' from the previous response.
+  string page_token = 6;
+}
+
+// This is the response from 'POST /expressionlevel/search' expressed as JSON.
+message SearchExpressionLevelsResponse {
+  // The list of matching quantifications.
+  repeated ExpressionLevel expression_levels = 1;
+
+  // The continuation token, which is used to page through large result sets.
+  // To get the next page of results, set this parameter to the value of
+  // 'nextPageToken' from the previous response.
+  string next_page_token = 2;
+}
+
+// ****************  /quantificationgroup/search  *******************
+// This request maps to the body of 'POST /quantificationgroup/search'
+// as JSON.
+message SearchQuantificationGroupsRequest {
+  // RNA Quantification to search.
+  string rna_quantification_id = 1;
+
+  // Quantification Groups of interest.
+  string quantification_group_id = 2;
```

Comment:
The fact that David needs to ask these questions means the documentation
needs more details ;-)

David Steinberg <notifications@github.com> writes:
> What purpose does specifying the quantification group ID serve here? The
> comment suggests that one can give multiple groups, but this is a singly valued
> field.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.*
>